internal/automanagement/manager.go

```diff
@@ -92,3 +92,14 @@ func (m *Manager) RunStream(ctx context.Context, req *backend.RunStreamRequest,
 	}
 	return status.Error(codes.Unimplemented, "unimplemented")
 }
+
+func (m *Manager) ProcessInstanceSettings(ctx context.Context, req *backend.ProcessInstanceSettingsRequest) (*backend.ProcessInstanceSettingsResponse, error) {
+	h, err := m.Get(ctx, req.PluginContext)
```

Comment:
Note. This feels wrong. To be able to handle create think needs to live outside manager or not try to get the instance from cache because that will cache a new instance even if the validation fails. 

Nit. To make things more explicit/easier for devs maybe worth splitting up create and update/delete where create doesn't have a plugin context and the latter does. That could be an abstraction in SDK contracts to ease developer API?

```go
func handle() {
  if create do something
  else if update do something
  else if delete do something
}
```

vs 

```go
func handleCreate() {
  do something
}

func handleUpdate) {
  do something
}

func handleDelete() {
  do something
}
```